supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -570,6 +620,58 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*:  xref:xplat[Microsoft Azure Cross-Platform Command-Line Interface]
 
+[discrete]
+[[ARM64]]
+==== image:images/yes.png[yes] ARM64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term for OpenShift Container Platform (OCP) attributes, Kubernetes, Operators, APIs, and CLI objects. Use this format in general sentences when referring to OCP features.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_Creating an ARM64 compute machine set._ 
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: xref:arm64[arm64]
+
+[discrete]
+[[arm64]]
+==== image:images/yes.png[yes] arm64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, and CLI objects. Use lowercase format and backticks when referring to objects or parameters.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_Valid values are `arm64`._
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: c
+
+[discrete]
+[[64-bit ARM]]
```

Comment:
```suggestion
[[64-bit-arm]]
```
The anchor in the brackets should be lowercase and hyphenated in place of spaces. Ditto other entries.

Reply:
Yes, @kelbrown20 . I do not see why not. You could describe the usage of the term by surrounding it in quotation marks. 

For example:

Use the term "64-bit ARM".

Reply:
As an example, see "Kubernetes master" in the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#_m). Just one example of many, but might go into a bit more detail about usage, but gives some sort of example.

Reply:
Oh I see how it works! Thank you!